### PR TITLE
Update SDK 15.5.1, add version labels, update app icon

### PR DESCRIPTION
**Pull request by duclong201**
[Jira ticket](https://bluedotinnovation.atlassian.net/jira/software/c/projects/BD/boards/42?modal=detail&selectedIssue=BD-4474)

### CODE REVIEW

File: BDPointAndroidXamarinDemo/BDPointAndroidXamarinDemo.csproj [hunk 1]
```diff
@@ -98,17 +94,21 @@
...
   <ItemGroup>
-    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.4.0" />
+    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.7.0.1" />
     <PackageReference Include="Xamarin.AndroidX.Work.Work.Runtime.Ktx">
-      <Version>2.5.0.1</Version>
+      <Version>2.7.1.5</Version>
     </PackageReference>
     <PackageReference Include="Bluedot.PointSDK.Android">
-      <Version>15.4.1</Version>
+      <Version>15.5.1.2-alpha</Version>
```
Comment (duclong201) [inline BDPointAndroidXamarinDemo/BDPointAndroidXamarinDemo.csproj:111]: This is currently using pre-release SDK version 15.5.1.2-alpha. Once it passed testing, and version 15.5.1 is release, I will update it to point to 15.5.1. Ticket created [here](https://bluedotinnovation.atlassian.net/browse/BD-4505?atlOrigin=eyJpIjoiMGZlYzg4OTVjYzdiNGNjNjgzYTE5ZDU2ZWMxN2IyY2QiLCJwIjoiaiJ9)

Comment (datkinnguyen) [reply BDPointAndroidXamarinDemo/BDPointAndroidXamarinDemo.csproj:111]: Makes sense that Min App is using a prerelease package 15.5.1.2-alpha for testing 👍

File: ReleaseNotes.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 #Release Notes
 
-- Update SDK 15.4.1
\ No newline at end of file
+- Update SDK 15.5.1
```
Comment (datkinnguyen) [inline ReleaseNotes.md:3]: Could you also mention Update Target Framework to Android 12?, for SDK release notes too.

Comment (duclong201) [reply ReleaseNotes.md:3]: Sure

File: BDPointAndroidXamarinDemo/Properties/AndroidManifest.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="io.bluedot.point.xamarin_demo" android:versionName="15.4.1" android:versionCode="5">
-	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="30" />
-	<application android:allowBackup="true" android:icon="@mipmap/ic_launcher" android:label="@string/app_name" android:roundIcon="@mipmap/ic_launcher_round" android:supportsRtl="true"></application>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="io.bluedot.point.xamarin_demo" android:versionName="1.0.0" android:versionCode="6">
```
Comment (duclong201) [inline BDPointAndroidXamarinDemo/Properties/AndroidManifest.xml:2]: Using 1.0.0 for less confusion with SDK version

**Pull request merged**